php/composer.json

```diff
@@ -18,7 +18,14 @@
         "slim/csrf": "^1.3",
         "ext-apcu": "*"
     },
+    "require-dev": {
+        "vimeo/psalm": "^5.25"
```

Comment:
not sure about this... seems like it breaks the twig lint workflow?

Reply:
bd9abf2 fix